app/src/main/res/layout/activity_migration.xml

```diff
@@ -6,7 +6,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/white_color">
+    android:background="@color/photonLightGrey05">
```

Comment:
Not too worried about mapping this since we are planning to remove the Fennec to Fenix migration.